### PR TITLE
Update libicu to 74.2 & rebuilds

### PR DESCRIPTION
**Pull request by joebonrichie**
**Summary**

libicu 74 & rebuilds

**Test Plan**

Install, no borky.

**Checklist**

- [x] Package was built and tested against unstable

### CODE REVIEW

**Comment by joebonrichie** [comment]
will reenable libicu support in libxml2 after landing this

File: packages/py/pyicu/package.yml [hunk 1]
```diff
@@ -1,8 +1,9 @@
 name       : pyicu
-version    : '2.11'
-release    : 28
+version    : '2.12'
+release    : 30
```
Comment (EbonJaeger) [inline packages/py/pyicu/package.yml:3]: Release double-bumped

**Review by EbonJaeger — CHANGES_REQUESTED** [review]
One package has a double-bump. Looks good, aside from 29 of the pspec files not having the packager information set. Not sure what we want to do about that.

**Comment by ReillyBrogan** [comment]
I'll fix-up the pspecs for this.

**Review by ReillyBrogan — APPROVED** [review]

**Pull request merged**